src/Components/Onboarding/ItemLink.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react"
+import styled, { StyledFunction } from "styled-components"
+
+import { fadeIn, fadeOut } from "../../Assets/Animations"
+import * as fonts from "../../Assets/Fonts"
+
+import Icon from "../Icon"
+
+const anchor: StyledFunction<State & React.HTMLProps<HTMLInputElement>> = styled.a
+const Link = anchor`
+  display: flex;
+  font-size: 14px;
+  color: black;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-family: ${fonts.primary.fontFamily};
+  border-top: 1px solid #e5e5e5;
+  &:hover {
+    background-color: #f8f8f8;
+  }
+  ${props => (props.fadeIn ? fadeIn : null)}
+  ${props => (props.fadeOut ? fadeOut : null)}
+`
+
+const Avatar = styled.img`
+  padding: 10px 15px 10px 10px;
+`
+
+const FullWidthCol = styled.div`
+  flex: 1;
+  display: flex;
+  align-items: center;
+`
+
+const Col = styled.div`
+  display: flex;
+  align-items: center;
+`
+
+export interface State {
+  fadeIn: boolean
+  fadeOut: boolean
+  item_id: string
+}
+
+interface Props extends React.HTMLProps<HTMLAnchorElement> {
+  item?: any
+  id: string
+  _id: string
+  name: string
+  image_url
```

Comment:
This property is missing a type. Interestingly that seems to default to `any`, was that the intent?

Reply:
I meant to type this as a string.